radixdlt/src/main/java/com/radixdlt/consensus/liveness/PacemakerImpl.java

```diff
@@ -36,19 +36,23 @@
  * Overly simplistic pacemaker
  */
 public final class PacemakerImpl implements Pacemaker, PacemakerRx {
-	static final int TIMEOUT_MILLISECONDS = 500;
+	static final int TIMEOUT_MILLISECONDS = 1000;
```

Comment:
Any specific reason for the increase? 
Adding a comment in source to explain the magic number would be sufficient for now